suitcase/utils/conftest.py

```diff
@@ -0,0 +1,123 @@
+import bluesky # noqa
+from bluesky.tests.conftest import RE # noqa
+from bluesky.plans import count # noqa
+from bluesky.plan_stubs import trigger_and_read, configure
+import event_model # noqa
+from ophyd.tests.conftest import hw # noqa
+import pytest # noqa
+from . import UnknownEventType # noqa
+
+
+# Some useful plans for use in testing
+def simple_plan(dets):
+    '''A simple plane which runs count with num=5'''
+    yield from count(dets, num=5)
```

Comment:
I think it makes sense to load up the meta-data with both reasonable place holders (user, beamline, etc) and some test specific stuff `SUITCASE_TEST_PLAN_NAME='simple_plan'`.

Reply:
good call, I am doing suitcase.json_metadata now where that is mroe relevant so I will make the change soon